src/webmon_app/plots/view_util.py

```diff
@@ -0,0 +1,19 @@
+from bs4 import BeautifulSoup
+
+
+def get_plotly_js(plot_data):
+    # The version of the plotly.js library to use
+    version = "2.9.0"
+    if plot_data:
+        soup = BeautifulSoup(plot_data, "html.parser")
+        plotly_div = soup.find("div", class_="plotly-graph-div")
+        if plotly_div and "data-plotlyjs-version" in plotly_div.attrs:
```

Comment:
It looks like `plot_publisher` adds the `plotlyjs-version` attribute not `data-plotlyjs-version`.